assets/lokomotive-kubernetes/bootkube/resources/bootstrap-manifests/bootstrap-controller-manager.yaml

```diff
@@ -18,6 +18,8 @@ spec:
     - --cluster-signing-cert-file=/etc/kubernetes/secrets/ca.crt
     - --cluster-signing-key-file=/etc/kubernetes/secrets/ca.key
     - --configure-cloud-routes=false
+    - --controllers=*,tokencleaner
```

Comment:
We don't specify token expiry time, so `tokencleaner` controller is not needed anyway?